canonical-websites/www.ubuntu.com
# Update Japanese UASD

## Done

- Updated Japanese UASD
- Archived old one, but not linked to
- Updated the old/new copy docs

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/ubuntu-advantage-service-description/ja](http://0.0.0.0:8001/legal/ubuntu-advantage-service-description/ja)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare it to the [copy doc](https://docs.google.com/document/d/1NHQS8MQxUhtAQZplcIFQP39qmrBOCTH508JpcEueSmI/edit#)

## Issue / Card

Fixes #4186
